R/tt_export.R

```diff
@@ -893,7 +894,7 @@ tt_to_flextable <- function(tt,
 }
 
 #' @describeIn tt_to_flextable main theme function for [export_as_docx()]
-#' @param font character(1). Defaults to `"Arial"`. If the font is not vailable, `flextable`
```

Comment:
Why was this not caught by the spelling?

Reply:
there is no spelling here as well 🤷

Reply:
this was caught by my pre-commit

Reply:
The spelling check, I think, should be there. It is annoying, but it is beneficial. @cicdguy I remember that we kind of decided to proceed with checks on PRs in rtables and formatters, but maybe we had some reasons to hold it off. Do you recall?